apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md

```diff
@@ -3,58 +3,29 @@ To use the Office and Office 365 multicolor brand icons, select the format and s
 Office brand icons come in predefined dimensions at 16px, 48px, and 96px. Although you may overwrite icon sizes with your custom styles, we recommend using our icons at the predefined sizes so the images appear as intended. Fabric includes custom media queries that will display our PNG brand icons in a different resolution depending on the device's pixel density. See the code sample below for an example:
 
 ```jsx
-// Sample code for displaying an Excel 96x96px Icon
-<div class="ms-BrandIcon--icon96 ms-BrandIcon--excel" />
+// Sample code for displaying an Word 96x96px Icon
+<div class="ms-BrandIcon--icon96 ms-BrandIcon--word" />
 ```
 
+### File type icons
+
+The most durable and future-proof way to get the right icon for a file or item is to use the [file-type-icons package from Fabric](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/file-type-icons).  Here's a simple [demo of how](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/FileTypeIcon/examples/FileTypeIcon.Basic.Example.tsx).  The code for the main method you'll use ([`getFileTypeIconProps`](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/file-type-icons/src/getFileTypeIconProps.ts)) is also well-commented. The icons are served up from Fabric-CDN but you should not directly clone or reference these.
```

Comment:
We should add that the npm package name is `@uifabric/file-type-icons`

Reply:
And ideally should include relevant docs inline rather than referring people to code comments